packages/tocco-util/src/form/reduxForm.js

```diff
@@ -64,11 +72,20 @@ export const entityToFormValues = entity => {
   return result
 }
 
+export const emptyValues = modelName => {
```

Comment:
I would prefer 

```
export const emptyValue = modelName => ({
  [wholeEntityField]: {
    entity: {
       model: modelName
    }
  }
})
```

The structure of the result object is immediately clear this way. But it's also ok if we leave it like this.